ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -892,11 +892,11 @@
             throttle_concurrent_builds_category:
                 - e2e-lock-per-testbed
             throttle_concurrent_builds_enabled: 'true'
-            status_context: jenkins-windows-containerd-conformance
+            status_context: jenkins-windows-containerd-with-ovs-conformance
             status_url: --none--
             success_status: Build finished.
-            failure_status: Failed. Add comment /test-windows-containerd-conformance to re-trigger.
-            error_status: Failed. Add comment /test-windows-containerd-conformance to re-trigger.
+            failure_status: Failed. Add comment /test-windows-containerd-with-ovs-conformance to re-trigger.
```

Comment:
Maybe this trigger phase is too long to be remembered, do you have any suggestions? @wenyingd

Reply:
If we don't support docker related test anymore, we may remove containerd and name it like `test-windows-with-ovs-conformance`
and if we don't have other `test-windows-conformance` test, maybe more 'ovs' as well.

Reply:
Yes, there is no reason to add `containerd` in trigger phase after removing docker jobs.

Reply:
Once docker job is removed, and support for running ovs user processes on host is deprecated  , it would be better to remove ovs from trigger phrase.

Reply:
The docker jobs have not been removed through this PR , the only jobs I have removed are containerd jobs for conformance and network policy tests since they are same as containerd-with-ovs jobs for the same

Reply:
@NamanAg30 We have another PR to remove docker job https://github.com/antrea-io/antrea/pull/5661 , you don't have to remove them in this PR.

Reply:
required changes have been made 
@XinShuYang

Reply:
@XinShuYang @NamanAg30 , since we are discussing about optimising containerd based jobs/names etc and workflows in the PR based on docker job removal in #5661 , we might want to make this PR depend on #5661 and define the sequence whether we want to merge #5661 first then this PR.